src/decompression/service/DecompressionService.ts

```diff
@@ -0,0 +1,18 @@
+import * as zlib from 'zlib';
+import * as Q from 'q';
+
+export default class DecompressionService {
+  public static decompress(value: string): Q.Promise<string> {
```

Comment:
Change this to return a native promise instead of a Q promise since it is a public function on a class that you are exporting in index.ts.